client/packages/common/src/intl/locales/en/distribution.json

```diff
@@ -60,10 +60,6 @@
   "label.requested-quantity": "Requested quantity",
   "label.select-rows-to-allocate-them": "Select rows to allocate them",
   "label.supply-quantity": "Quantity to supply",
-  "label.unit_one": "{{unit}}",
```

Comment:
couldn't find this label.unit_one in the es translation file so have deleted from all